lib/components/Editor/CustomExtensions/Mention/utils.js

```diff
@@ -1,11 +1,24 @@
+import Fuse from "fuse.js";
+import { pluck } from "ramda";
+
 export const formatMentions = (users = []) =>
   users.sort((user1, user2) => user1.name.localeCompare(user2.name));
 
 export const createMentionSuggestions = (users = []) => {
   const mentions = formatMentions(users);
 
-  return ({ query }) =>
-    mentions.filter(({ name }) =>
-      name.toLocaleLowerCase().includes(query.toLocaleLowerCase())
-    );
+  return ({ query }) => {
+    if (!query) {
+      return mentions;
```

Comment:
```suggestion
      return mentions.slice(0, 10);
```

@gaagul _a merge after making this minor change.